lib/Parse/ParseDecl.cpp

```diff
@@ -5759,7 +5759,6 @@ ParserStatus Parser::parseGetSet(ParseDeclOptions Flags,
 
       // parsingLimitedSyntax mode cannot have a body.
       if (parsingLimitedSyntax) {
-        diagnose(Tok, diag::expected_getset_in_protocol);
         Invalid = true;
```

Comment:
I feel uncomfortable with marking `Invalid` without emitting diagnostics. Could you just leave this diagnostics as is for now? I know 
```swift
protocol Protocol9 {
   var a: Int { return 0 }
 }
```
results kind of duplicated errors (`expected get or set in a protocol property` and `property in protocol must have explicit { get } or { get set } specifier`) But still.

Reply:
Yep! I reverted the change :)